### DISCUSSION TITLE
Quarto figure layout

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#9824] [Q&A]
### Description

How can I have a figure layout like this?:

figure1
                figure3
figure2

This mean, 2 rows, 2 columns, but in the second column the figure is centered.

**Comment by mcanouil** [comment]
Did you look at the documentation about figures? There is a whole guide: https://quarto.org/docs/authoring/figures.html#complex-layouts

**Comment by ManuelSpinola** [comment]
Thank you very much. Yes, I tried this but it does not center the 3rd figure.

::: {layout="[[1,1], [1]]" layout-valign="center"}
![Surus](surus.png)

![Hanno](hanno.png)

![Lin Wang](lin-wang.png)
:::

**Reply by mcanouil** [reply]
Please format properly your code using markdown code blocks and share reproducible example.

Note that you are using vertical alignment and not horizontal alignment.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by cscheid** [comment]
That's not possible with Quarto. Our layout system is "rows-first", and is not going to change anytime soon.

With that said, you can always wrap your figures in a containing div, add appropriate classes, and then handle the layout entirely manually by yourself.

**Comment by ManuelSpinola** [comment]
Thank you very much.
